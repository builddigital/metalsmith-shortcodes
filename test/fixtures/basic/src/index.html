---
date: 2014-03-29
---

<p>Here's foo!</p>

[foo klass=bar]
