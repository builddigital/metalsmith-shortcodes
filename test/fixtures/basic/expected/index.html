
<p>Here's foo!</p>

<div class="bar"><foo data-date="2014-03-29"></foo></div>
